python/cudf_kafka/cudf_kafka/_lib/kafka.pyx

```diff
@@ -1,7 +1,8 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
+cimport cpython
 from libc.stdint cimport int32_t, int64_t
-from libcpp cimport bool
+from libcpp cimport bool, nullptr
```

Comment:
This is out of scope for this PR so you can skip if you want, but we should probably `cimport bool as cbool` or something else to avoid shadowing Python's bool in this scope. That could lead to confusing errors later.